articles/libraries/auth0-swift/index.md

```diff
@@ -28,19 +28,17 @@ To install it, simply add the following line to your Podfile:
 pod 'Auth0', '~> 1.0'
 ```
 
-### Carthage
+### Using Carthage
 
 In your Cartfile add this line
 
 ```
 github "auth0/Auth0.swift" ~> 1.0
 ```
 
-## Usage
+## Auth0.plist
```

Comment:
I don't think the list it'self should be the header, perhaps 'Credentials' as the header and 'Auth0.plist' as a sub header.